Build failure: tagutil

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

`nix-build -A tagutil`

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/308138074

### Relevant log output

```console
Running phase: unpackPhase
unpacking source archive /nix/store/w6qh5lkfna1dkhw9mknhdfli1ilvxzz6-source
source root is source/src
Running phase: patchPhase
substituteStream() in derivation tagutil-3.1: WARNING: '--replace' is deprecated, use --replace-{fail,warn,quiet}. (file 'CMakeLists.txt')
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/wqpgq9c0f0wbkb16qzws14kfpj7r1my0-tagutil-3.1/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/wqpgq9c0f0wbkb16qzws14kfpj7r1my0-tagutil-3.1/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/wqpgq9c0f0wbkb16qzws14kfpj7r1my0-tagutil-3.1/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/wqpgq9c0f0wbkb16qzws14kfpj7r1my0-tagutil-3.1/share/doc/tagutil -DCMAKE_INSTALL_INFODIR=/nix/store/wqpgq9c0f0wbkb16qzws14kfpj7r1my0-tagutil-3.1/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/wqpgq9c0f0wbkb16qzws14kfpj7r1my0-tagutil-3.1/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/wqpgq9c0f0wbkb16qzws14kfpj7r1my0-tagutil-3.1/include -DCMAKE_INSTALL_SBINDIR=/nix/store/wqpgq9c0f0wbkb16qzws14kfpj7r1my0-tagutil-3.1/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/wqpgq9c0f0wbkb16qzws14kfpj7r1my0-tagutil-3.1/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/wqpgq9c0f0wbkb16qzws14kfpj7r1my0-tagutil-3.1/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/dmypp1h4ldn0vfk3fi6yfyf5yxp9yz0k-gcc-wrapper-14.3.0/bin/strip -DCMAKE_RANLIB=/nix/store/dmypp1h4ldn0vfk3fi6yfyf5yxp9yz0k-gcc-wrapper-14.3.0/bin/ranlib -DCMAKE_AR=/nix/store/dmypp1h4ldn0vfk3fi6yfyf5yxp9yz0k-gcc-wrapper-14.3.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/wqpgq9c0f0wbkb16qzws14kfpj7r1my0-tagutil-3.1
CMake Error at CMakeLists.txt:2 (cmake_minimum_required):
  Compatibility with CMake < 3.5 has been removed from CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.


-- Configuring incomplete, errors occurred!
```

### Additional context

Related to CMake 4.1.1 update: #445447
Upstream PR has been made which could be upstreamed: https://github.com/kaworu/tagutil/pull/61

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.16.9, NixOS, 25.11 (Xantusia), 25.11.20250928.e9f00bd`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.5`
 - nixpkgs: `/nix/store/jc3mfa7ybzh32bkvdj5xiib1vkx8jy6x-source`

### Notify maintainers



---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/ta/tagutil/package.nix
+++ b/pkgs/by-name/ta/tagutil/package.nix
@@ -13,13 +13,13 @@
 
 stdenv.mkDerivation rec {
   pname = "tagutil";
-  version = "3.1";
+  version = "3.1-unstable-2025-06-16";
 
   src = fetchFromGitHub {
     owner = "kaworu";
     repo = "tagutil";
-    rev = "v${version}";
-    sha256 = "sha256-oY1aGl5CKVtpOfh8Wskio/huWYMiPuxWPqxlooTutcw=";
+    rev = "c8b20ef350b1a8a67a747590e2e88b41f802cce4";
+    sha256 = "sha256-sKnBS9kXhJ2atN6A3qcX9A+0A7WfNkOe+nKSblL3i0o=";
   };
 
   sourceRoot = "${src.name}/src";

